File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans;
        int n = nums.size();
        ans.resize(2 * n);
        for (int i = 0; i < n; i++)
        {
            int j = i + n;
            ans[i] = nums[i];
            ans[j] = nums[i];
        }
        return (ans);
    }
};

File: Data Structures & Algorithms/maximum-subarray/submission-0.cs
public class Solution {
    public int MaxSubArray(int[] nums) {
        var max = nums[0];
        var cur = 0;

        foreach (var n in nums) {
            cur += n;
            if (cur > max) {
                max = cur;
            }

            if (cur < 0) {
                cur = 0;
                continue;
            }
        }

        return max;
    }
}
